custom/icds_reports/static/js/directives/info-message/info-message.directive.js

```diff
@@ -29,6 +32,18 @@ function InfoMessageController($location) {
             vm.currentMonth = moment().format("MMMM");
             vm.nextMonth = moment().startOf('month').add(1, 'months').format("MMMM");
             vm.year = moment().format('YYYY');
+            vm.yearOfPreviousMonth = vm.year;
+            vm.yearOfNextMonth = vm.year;
+            if (currentMonth === 1) {
+                vm.yearOfPreviousMonth = moment().startOf('year').subtract(1, 'years').format('YYYY');
+            }
+            vm.yearOfTwoMonthsAgo = vm.yearOfPreviousMonth;
```

Comment:
oh I see how you're doing it. I guess this is ok, though is more confusing of a workflow for my brain.

Reply:
yearOfTwoMonthsAgo value will be based on yearOfPreviousMonth. So assigned it after yearOfPreviousMonth..so had to do like this